interop/soak_tests.go

```diff
@@ -0,0 +1,204 @@
+/*
+ *
+ * Copyright 2014 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// client starts an interop client to do rpc_soak test and channel_soak test.
+//
+// See interop test case descriptions [here].
+//
+// [here]: https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md
```

Comment:
I'm surprised we don't have a linter that would complain about this?

1. There should only be one package comment per package, and that should typically go into a more general file.  If one doesn't already exist, you shouldn't add one here.

2. Package comments must begin with `Package <foo> [does xyz]`.

This effectively is already present in `test_utils.go` so please just delete this.